modules/test/playwright/tests/frontend-data-set-web/main/pages/FDSSamplePage.ts

```diff
@@ -30,6 +30,7 @@ export class FDSSamplePage {
 	readonly table: {
 		bodyRows: Locator;
 		container: Locator;
+		firstColumnHeader: Locator;
```

Comment:
:broom: 
is this one needed? it seems a duplication of what's in `this.table` isn't it?